db/migrate/20190711135901_add_fields_to_proceeding_types.rb

```diff
@@ -0,0 +1,9 @@
+class AddFieldsToProceedingTypes < ActiveRecord::Migration[5.2]
+  def change
+    add_column :proceeding_types, :default_level_of_service_id, :integer
+    add_column :proceeding_types, :default_level_of_service_name, :string
+    add_column :proceeding_types, :default_cost_limitation_delegated_functions, :integer
+    add_column :proceeding_types, :default_cost_limitation_substantive, :integer
```

Comment:
Another two attributes sharing a common prefix, `default_cost_limitation_` in this case. However, I expect that the two are not closely coupled ( that is, that two proceeding types could share the same `default_cost_limitation_delegated_functions` and still have different `default_cost_limitation_substantive` values.

However, if that is not the case (that is, if a proceeding type has a `default_cost_limitation_delegated_functions` value of X, it will always have a `default_cost_limitation_substantive` value of Y), then a separate object should be considered.

Reply:
Of course, if default cost limitations are tightly coupled to the service level, they would belong in `ServiceLevel`

Reply:
They are not related to service level, and they are not tightly coupled. - they simply represent the default cost limitation for each type of applicaiton: a substantive application and a delegated function application.  So I think these are best left as-is.

Reply:
Agreed, I'll leave this one alone.